index.html

```diff
@@ -201,29 +201,36 @@ <h2>ZCAP-LD by Example</h2>
              practice in general (outside of DIDs) but will make the example
              noisier -->
         <pre class="example highlight javascript">
-    {"@context": ["https://example.org/zcap/v1",
-                  "https://autopower.example/"],
-     "id": "https://whatacar.example/a-fancy-car/proc/7a397d7b",
-
-     // Since this is the first delegated capability, the parentCapability
-     // points to the target this capability will operate against
-     // (in this case, Alyssa's Car)
-     "parentCapability": "https://whatacar.example/a-fancy-car",
-
-     // We are granting authority specifically to one of Alyssa's
-     // cryptographic keys (not to be confused with the car
-     // key metaphor!)
-     "invoker": "https://social.example/alyssa/#key-for-car",
-
-     // Finally we sign this object with cryptographic material from
-     // Alyssa's Car's capabilityDelegation field, and using the
-     // capabilityDelegation proofPurpose.
-     "proof": {
-        "type": "RsaSignature2016",
-        "proofPurpose": "capabilityDelegation",
-        "created": "2016-02-08T16:02:20Z",
-        "creator": "https://whatacar.example/a-fancy-car#key-1",
-        "signatureValue": "IOmA4R7TfhkYTYW8...CBMq2/gi25s="}}
+{
+  "@context": ["https://w3id.org/security/v2",
+               "https://autopower.example/"],
+
+  "id": "https://whatacar.example/a-fancy-car/proc/7a397d7b",
+
+  // Since this is the first delegated capability, the parentCapability
+  // points to the target this capability will operate against
+  // (in this case, Alyssa's Car)
+  "parentCapability": "https://whatacar.example/a-fancy-car/caps#1",
```

Comment:
Hmm, don't quite know if this is correct. It might not be the right intent. The parentCapability is the car, not the capability... though I understand how this might be confusing.

Reply:
agree, if this is the first capability from a controller, the parent should be the controller, not a verification method or capability.

Reply:
Yeah, and I expect that the intent here (per the comment in the code example) *is* the controller, which is the car. I'm going to merge my suggestion and then merge this into the spec... 'cause we want to fix the links.